storage/storage.go

```diff
@@ -1588,6 +1588,11 @@ type Query struct {
 	// which returns all properties. Passing ProjectionNoACL will omit Owner and ACL,
 	// which may improve performance when listing many objects.
 	Projection Projection
+
+	// If true, objects that end in exactly one instance of delimiter have
+	// their metadata included in items[] in addition to the relevant part of
+	// the object name appearing in prefixes[].
+	IncludeTrailingDelimiter bool
 }
```

Comment:
Can you also write an integration test to verify that this behaves as expected with real objects? Example for Query.Projection is here: https://github.com/googleapis/google-cloud-go/blob/9f79ac4f5ff32fc99fb3d49ec59cf7dd87ce21f9/storage/integration_test.go#L1902

Reply:
I read through the [integration test scrolls](https://github.com/googleapis/google-cloud-go/blob/main/CONTRIBUTING.md#testing) but the evil wizard frightened me.  I tried again with some magical potions 🍷 but this quest seems to require some serious XP.  Is there a cheat code to avoid grinding?

Reply:
Added test.